src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/restapi/controller/CourseController.kt

```diff
@@ -134,4 +135,20 @@ constructor(
 
     return ResponseEntity.ok(savedCourse.toApi())
   }
+
+  override fun createCourse(courseCreateRequest: CourseCreateRequest): ResponseEntity<Course> {
+    val course = CourseEntity(
+      name = courseCreateRequest.name,
+      identifier = courseCreateRequest.identifier,
+      description = courseCreateRequest.description,
+      alternateName = courseCreateRequest.alternateName,
+      audience = courseCreateRequest.audience,
```

Comment:
Validation on audience too (use the get audiences method that already exists to get the audience names and then use this to populate the colour)